.travis.yml

```diff
@@ -10,11 +10,13 @@ addons:
     packages:
     - libgconf-2-4
 
-# cache both yarn and Cypress binary
+# cache both yarn and Cypress binar
+# also cache TypeScript builds, this only rebuilds when code changes
 cache:
   yarn: true
   directories:
     - ~/.cache
+    - .build-info
```

Comment:
Will this work? The documentation wasn't clear about subdirectories. Does `.build-info` match `**/.build-info`?

Reply:
Assuming this does work for Travis, I don't know much about how a TS build cache works. From what I can tell, this determines a graph for both type checking _and_ emitting. It looks like we emit code using Typescript. For this to work correctly, don't we also need to cache `dist` folders? I think timestamps and versions are fine for type-checking as type-checks have _no output_, but emitting does have an output.